app/presenters/check_answers/base.rb

```diff
@@ -0,0 +1,19 @@
+module CheckAnswers
+  class Base
+    def translate_table_key(table, key, **opt)
+      I18n.t("steps.check_answers.show.sections.#{table}.#{key}.key", **opt)
```

Comment:
so we have a separate `summary.yml` that I created for to store the translations for the summary page... I lean towards having these either in that file or their own. :)